web/tests/echo.test.cjs

```diff
@@ -34,6 +34,10 @@ const message_store = mock_esm("../src/message_store", {
 
     update_booleans() {},
 
+    update_message_content(message, new_content) {
```

Comment:
I'd appreciate other folks helping think carefully about if there's anywhere else need to switch to calling this